datasets/universal_dependencies/universal_dependencies.py

```diff
@@ -0,0 +1,1313 @@
+        "dev": "UD_Turkish-IMST/r2.7/tr_imst-ud-dev.conllu",
+        "test": "UD_Turkish-IMST/r2.7/tr_imst-ud-test.conllu",
+    },
+    "tr_pud": {
+        "test": "UD_Turkish-PUD/r2.7/tr_pud-ud-test.conllu",
+    },
+    "uk_iu": {
+        "train": "UD_Ukrainian-IU/r2.7/uk_iu-ud-train.conllu",
+        "dev": "UD_Ukrainian-IU/r2.7/uk_iu-ud-dev.conllu",
+        "test": "UD_Ukrainian-IU/r2.7/uk_iu-ud-test.conllu",
+    },
+    "hsb_ufal": {
+        "train": "UD_Upper_Sorbian-UFAL/r2.7/hsb_ufal-ud-train.conllu",
+        "test": "UD_Upper_Sorbian-UFAL/r2.7/hsb_ufal-ud-test.conllu",
+    },
+    "ur_udtb": {
+        "train": "UD_Urdu-UDTB/r2.7/ur_udtb-ud-train.conllu",
+        "dev": "UD_Urdu-UDTB/r2.7/ur_udtb-ud-dev.conllu",
+        "test": "UD_Urdu-UDTB/r2.7/ur_udtb-ud-test.conllu",
+    },
+    "ug_udt": {
+        "train": "UD_Uyghur-UDT/r2.7/ug_udt-ud-train.conllu",
+        "dev": "UD_Uyghur-UDT/r2.7/ug_udt-ud-dev.conllu",
+        "test": "UD_Uyghur-UDT/r2.7/ug_udt-ud-test.conllu",
+    },
+    "vi_vtb": {
+        "train": "UD_Vietnamese-VTB/r2.7/vi_vtb-ud-train.conllu",
+        "dev": "UD_Vietnamese-VTB/r2.7/vi_vtb-ud-dev.conllu",
+        "test": "UD_Vietnamese-VTB/r2.7/vi_vtb-ud-test.conllu",
+    },
+    "wbp_ufal": {
+        "test": "UD_Warlpiri-UFAL/r2.7/wbp_ufal-ud-test.conllu",
+    },
+    "cy_ccg": {
+        "train": "UD_Welsh-CCG/r2.7/cy_ccg-ud-train.conllu",
+        "test": "UD_Welsh-CCG/r2.7/cy_ccg-ud-test.conllu",
+    },
+    "wo_wtb": {
+        "train": "UD_Wolof-WTB/r2.7/wo_wtb-ud-train.conllu",
+        "dev": "UD_Wolof-WTB/r2.7/wo_wtb-ud-dev.conllu",
+        "test": "UD_Wolof-WTB/r2.7/wo_wtb-ud-test.conllu",
+    },
+    "yo_ytb": {
+        "test": "UD_Yoruba-YTB/r2.7/yo_ytb-ud-test.conllu",
+    },
+}
+
+
+class UniversaldependenciesConfig(datasets.BuilderConfig):
+    """BuilderConfig for Universal dependencies"""
+
+    def __init__(self, data_url, **kwargs):
+        super(UniversaldependenciesConfig, self).__init__(version=datasets.Version("2.7.0", ""), **kwargs)
+
+        self.data_url = data_url
+
+
+class Universaldependencies(datasets.GeneratorBasedBuilder):
+    VERSION = datasets.Version("2.7.0")
+    BUILDER_CONFIGS = [
+        UniversaldependenciesConfig(
+            name=name,
+            description=_DESCRIPTIONS[name],
+            data_url="https://github.com/UniversalDependencies/" + _UD_DATASETS[name]["test"].split("/")[0],
+        )
+        for name in _NAMES
+    ]
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "idx": datasets.Value("string"),
+                    "text": datasets.Value("string"),
+                    "tokens": datasets.Sequence(datasets.Value("string")),
+                    "lemmas": datasets.Sequence(datasets.Value("string")),
+                    "upos": datasets.Sequence(
```

Comment:
maybe rename this column pos_tags ?

Reply:
For these datasets, I prefer to keep the original name, it is less confusing because here you have multiple sort of POS tags.